opentech/apply/projects/forms.py

```diff
@@ -20,3 +22,20 @@ def __init__(self, instance=None, user=None, *args, **kwargs):
     def save(self, *args, **kwargs):
         submission = self.cleaned_data['submission']
         return Project.create_from_submission(submission)
+
+
+class UpdateProjectLeadForm(forms.ModelForm):
+    class Meta:
+        fields = ['lead']
+        model = Project
+
+    def __init__(self, user=None, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        lead_field = self.fields['lead']
+        lead_field.label = f'Update lead from {self.instance.lead} to'
+
+        qwargs = Q(groups__name=STAFF_GROUP_NAME) | Q(is_superuser=True)
+        lead_field.queryset = (lead_field.queryset.exclude(pk=self.instance.lead_id)
```

Comment:
The underlying DB field is Nullable - do you need to tweak the settings on the form field to prevent empty response or am i over thinking it?

Reply:
`lead_field.queryset` is a QS of User so this should be fine.  If `self.instance.lead_id` is `None` then we're just excluding User's with `pk=None`.